content/docs/2.8/concepts/authentication.md

```diff
@@ -93,6 +93,7 @@ metadata:
 spec:
   podIdentity:
       provider: none | azure | azure-workload | aws-eks | aws-kiam  # Optional. Default: none
+      identityID: <identity-id>                         # Optional. Only used by azure & azure-workload providers.
```

Comment:
Nit, we typically use `Id` instead of `ID`
```suggestion
      identityId: <identity-id>                         # Optional. Only used by azure & azure-workload providers.
```

Reply:
Same for other places, obviously.